### PR TITLE
chore: copy changes

### LINKED ISSUE #4630: [React] Response ID instead of response number in individual respondent view
image19

### LINKED ISSUE #4632: Copy changes for toast
Change copy to "Your form is closed to new responses"
Screenshot 2022-08-22 at 5 23 04 PM

change the copy to “Your form is now open. If you expect a large number of responses, AutoArchive your mailbox to avoid losing any of them.” 
Screenshot 2022-08-22 at 5 23 46 PM

### LINKED ISSUE #4642: OTP login copy changes
Before OTP - label, only, no description: “Log in with a .gov.sg or other whitelisted email address”
After OTP - label only, no description: “Enter OTP sent to [user@agency.gov.sg](mailto:user@agency.gov.sg)”  (email is admin’s email they entered, lowercased)
Screenshot 2022-08-22 at 5 26 24 PM

### PR DESCRIPTION
**Pull request by tshuli**
## Problem
- Simple PR to update various copy
- Closes #4632
- Closes #4630
- Closes #4642

## Screenshots for #4642 
- Before:

Screenshot 2022-08-23 at 9 07 56 AM
Screenshot 2022-08-23 at 9 06 25 AM

- After:
Screenshot 2022-08-23 at 9 00 47 AM
Screenshot 2022-08-23 at 9 05 57 AM

### CODE REVIEW

File: frontend/src/features/login/components/OtpForm.tsx [hunk 1]
```diff
@@ -42,12 +42,8 @@ export const OtpForm = ({
...
       <FormControl isInvalid={!!formState.errors.otp} mb="2.5rem">
-        <FormLabel
-          isRequired
-          htmlFor="otp"
-          description={`Enter OTP sent to ${email}`}
-        >
-          One-time password
+        <FormLabel isRequired htmlFor="otp">
+          {`Enter OTP sent to ${email.toLowerCase()}`}
         </FormLabel>
```
Comment (justynoh) [inline frontend/src/features/login/components/OtpForm.tsx:47]: actually this line should be localized too right haha

**Review by justynoh — APPROVED** [review]
lgtm

File: frontend/src/features/login/components/OtpForm.tsx (hunk 1 above)
Comment (tshuli) [reply frontend/src/features/login/components/OtpForm.tsx:47]: Thanks! Actually not just this, the otp form as a whole is not localised 🤔 (only Login Form is localised)
I think expanding localisation is outside the scope of this PR though so I'll leave it for now 😅

**Comment by timotheeg** [comment]
@tshuli can we merge this?

**Comment by tshuli** [comment]
@timotheeg yes, thanks! was pending design review previously which has now been approved.

**Pull request merged**